inst/Upgrades.qml

```diff
@@ -71,7 +71,522 @@ Upgrades
 		{
 			name:		"restrictedSE"
 			condition:	function(options) { return options["restrictedSE"] === "none"; }
-			jsValue:	"standard"
+			jsonValue:	"standard"
 		}
 	}
+
+	// Option renaming for syntax
+	Upgrade
+	{
+		functionName:		"Anova"
+		fromVersion:		"0.16.3"
+		toVersion:			"0.16.4"
+
+		// Display.qml
+		ChangeRename {	from: "VovkSellkeMPR";	to: "vovkSellke"	}
+
+		// AssumptionChecks.qml
+		ChangeRename {	from: "homogeneityNone";	to: "homogeneityCorrectionNone"		}
+		ChangeRename {	from: "homogeneityBrown";	to: "homogeneityCorrectionBrown"	}
+		ChangeRename {	from: "homogeneityWelch";	to: "homogeneityCorrectionWelch"	}
+
+		// Contrasts.qml
+		ChangeRename {	from: "confidenceIntervalsContrast";			to: "contrastCi"		}
+		ChangeRename {	from: "confidenceIntervalIntervalContrast";		to: "contrastCiLevel"	}
+
+		// OrderRestrictions.qml
+		ChangeRename {	from: "restrictedIncludeIntercept";					to: "restrictedInterceptInclusion"					}
+		ChangeRename {	from: "restrictedModelShowAvailableCoefficients";	to: "restrictedAvailableCoefficients"				}
+		ChangeRename {	from: "restrictedModelSummaryByDefault";			to: "restrictedModelSummaryForAllModels"			}
+		ChangeRename {	from: "restrictedMarginalMeansByDefault";			to: "restrictedMarginalMeanForAllModels"			}
+		ChangeRename {	from: "restrictedInformedHypothesisTestByDefault";	to: "restrictedInformedHypothesisTestForAllModels"	}
+
+		// restrictedModels is an array of option lists so we need to change the option names for each model
+		ChangeJS
+		{
+			name: "restrictedModels"
+			jsFunction: function(options)
+			{
+				let newModels = options["restrictedModels"].map(model => {
+					let newModel = {};
+					newModel.informedHypothesisTest	= model.informedHypothesisTest;
+					newModel.marginalMean			= model.marginalMeans;
+					newModel.summary				= model.modelSummary;
+					newModel.name					= model.modelName;
+					newModel.syntax					= model.restrictionSyntax;
+
+					return newModel ;
+				});
+
+				return newModels;
+			}
+		}
+
+		ChangeRename {	from: "restrictedModelComparisonHighlightCoefficients";		to: "restrictedModelComparisonCoefficientsHighlight"	}
+
+		ChangeRename {	from: "restrictedSE";	to: "restrictedHeterogeneityCorrection"	}
+
+		ChangeJS
+		{
+			name: "restrictedHeterogeneityCorrection"
+			jsFunction: function(options)
+			{
+				switch(options["restrictedHeterogeneityCorrection"])
+				{
+					case "standard":	return "none";
+					case "HC0":			return "huberWhite0";
+					case "HC1":			return "huberWhite1";
+					case "HC2":			return "huberWhite2";
+					case "HC3":			return "huberWhite3";
+					case "HC4":			return "huberWhite4";
+					case "HC4m":		return "huberWhite4m";
+					case "HC5":			return "huberWhite5";
+					default:			return options["restrictedHeterogeneityCorrection"];
+				}
+			}
+		}
```

Comment:
@JorisGoosen since the upgrade 0.16.3 -> 0.16.4 already does this, I don't have to do https://github.com/jasp-stats/INTERNAL-jasp/issues/1911#issuecomment-1194000000, right?

That way I would prefer to keep the 0.15 -> 0.16.3 upgrade anyway as it at least documents what kind of shenanigans happened with this option between the versions, even though the upgrade does not work in 0.16.3.

Reply:
Yeah this should also work!